contract.md

```diff
@@ -12,29 +27,42 @@ like to work on, as part of your project (for example, other things related to D
 you have come across, or topics from other classes)? Why are you interested in working on
 these things?_
 
+The concepts from class that I will be using are natural-language syntax, pattern matching, language mapping, and the idea of building a bridge between the dsl and the host language. Some things that we have not covered in class that I will be taking from previos expirences will be things such as data structure creatation/implamentation depending on libraries avaliable and API usage such as sending data to a live excell sheet. I think all of these skills will build a very good project. Im excited to dive deeper into the natural-language syntax/semi-natural-language syntax. I think this will be the easiest way for all users to utalize the dsl. Also it will be important that my language maps out to the data structure that represents a stat sheet. 
+
 ## Time management plan
 
 _How do you plan to set aside time outside of class, to work on the project? Are there
 intermediate milestones that you can create, to help you make consistent progress?_
 
+I plan to work on the project on Mondays, Wensdays, Fridays and Sundays. On Monday I will use the time in class to finish the critique for people in my group. Additionally if that is not finished in class then I will finish it later that night. Monday night I will plan to either finsh the critique or start working on the next section of my project. On Wensdays I will use the time in class to work on the project and ask questions on things that I may be having trouble with/ need help with. Wensday night I will plan to continue the section of my project that I have been working on that week for and hour or two. On Fridays I will work on the project in the later afternoon for a few hours before dinner. On Sunday I plan to work on my project for a few hours and then complete the Project notebook for the week. I plan to split the project into 5 sections of worked based off of my goals. This way I can devide the five weeks into an even set of work for the project and each goal will have rougly 1 week of time to complete.
+
```

Comment:
I like that you clearly outlined your plan for each day of the week, I'd be cautious to ensure that you hold yourself accountable because while it is good to not have long blocks when you plan to work since missing one day isn't a huge setback. However, it could be difficult to get set up, and getting started each of these days could bite into your capacity for development.